_posts/en/newsletters/2020-03-11-newsletter.md

```diff
@@ -0,0 +1,146 @@
+  abandon the proposal.  As we've mentioned [previously][segwit
+  signmessage], there is currently no widely adopted way for wallets to
+  create and verify signed messages for anything besides legacy P2PKH
+  addresses.  If wallet developers want to enable this capability for
+  P2SH, P2WPKH, P2WSH, and (if taproot is activated) P2TR addresses,
+  they are recommended to review Alm's email and provide feedback on
+  their preferred path forward.
+
+## Bitcoin Core PR Review Club
+
+The [Bitcoin Core PR Review Club][] is a weekly IRC meeting for newer
+contributors to the Bitcoin Core project to learn about the review process. An
+experienced Bitcoin Core contributor provides background information about a
+selected PR and then leads a discussion on IRC.
+
+The Review Club is an excellent way to learn about the Bitcoin protocol, the
+Bitcoin Core reference implementation, and the process of making changes to
+Bitcoin. Notes, questions and meeting logs are posted on the website for those
+who are unable to attend in real time, and as a permanent resource for those
+wishing to learn about the Bitcoin Core development process.
+
+In this section, we summarise a recent Review Club meeting.
+
+- **[Try to preserve outbound block-relay-only connections during restart][review club 17428]:**
+  This PR ([#17428][Bitcoin Core #17954]) by Hennadii Stepanov adds the concept
+  of _anchor connections_ to Bitcoin Core, which the node will preferably try to
+  reconnect to between restarts. These persistent connections could mitigate some
+  classes of [eclipse attacks][topic eclipse attacks].
+
+    Discussion began by establishing some fundamental concepts of eclipse attacks:
+
+    - Q: give a quick explanation of what an eclipse attack is?
+
+        _A: when a node is isolated from all honest peers_
+
+    - Q: describe how an adversary would eclipse a node?
+
+        _A: fill up their address list with addresses the attacker owns, then
+        force them to restart or wait for them to restart_
+
+    - Q: if a node is eclipsed, what sort of attacks can an adversary execute on the
+      victim?
+
+        _A: withholding blocks, censoring transactions, and
+        de-anonymizing transaction sources_.
+
+    Then changes in the PR were analyzed:
+
+    - Q: how does this PR mitigate an eclipse attack?
+
+        _A: by keeping a
+        list of some of the nodes you were connected to (anchor connections) and then
+        reconnecting to them on restart_
+
+    - Q: what are the conditions for a peer to become an anchor?
+
+        _A: the peer must be a blocks-only peer_.
+
+    Later in the meeting, there was discussion about the trade-offs and design
+    decisions in the PR:
+
+    - Q: why are only blocks-relay-only peers used as anchors?
+
+        _A: to make network topology inference harder and preserve network
+        privacy_
+
+    - Q: what happens if you choose an anchor who is able to remote-crash your
```

Comment:
perhaps s/who/that/ (since it is a non-human peer object like myself)

Reply:
fixed